test/contract based tests/EnvironmentTests.js

```diff
@@ -27,6 +28,40 @@ describe('Lossless Environment', () => {
             env.reportingAmount,
           );
       });
+
+      it('should not revert when setting the rewards', async () => {
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setLosslessReward(50),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setStakersReward(45),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setCommitteeReward(1),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setReporterReward(4),
+        ).to.not.be.reverted;
+      });
+
+      it('should get the rewards correctly', async () => {
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setLosslessReward(50),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setStakersReward(45),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setCommitteeReward(1),
+        ).to.not.be.reverted;
+        await expect(
+          env.lssReporting.connect(adr.lssAdmin).setReporterReward(4),
+        ).to.not.be.reverted;
+
+        expect(
+          await env.lssReporting.getRewards(),
```

Comment:
I would recommend verifying that returned values are correct.

Reply:
Added on ee22d4a